pkg/manifest/helm_processor.go

```diff
@@ -262,26 +281,33 @@ func (h *helm) IsConsistent(stringifedManifest string, deployInfo types.InstallI
 		return false, err
 	}
 
+	for i := range postRuns {
```

Comment:
Basically two concerns for this PostRun hook:
1. The time it get triggered, should it run at the end for the place it get introduced? For example, this `IsConsistent` check, the PostRun should run after final ` h.repoHandler.Update(ctx)`, or?

2. We offer same PostRun for install/uninstall/consistency check, how we distinguish which PostRun for which case? should we offer different PostRun options?

Reply:
The reason to have postrun is just a generic hook in which one can remove resources. The reason why I put it there is because the verification of the resources happens after the post run. so it's possible to influence the check ready state with the post run. If we don't need / want this I can put them at the very end.